SPECS/vim/vim.spec

```diff
@@ -2,13 +2,14 @@
 Summary:        Text editor
 Name:           vim
 Version:        9.0.2190
-Release:        2%{?dist}
+Release:        3%{?dist}
```

Comment:
You missed updating the changelog. Also, if we're only adding RPM macros, I'd consider creating a separate subpackage, since the macros may not be useful during run-time (we save image space) and the default package may not be necessary during build time.

Reply:
Updated changelog and moved this into subpackage